CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## Version 5.5.0
+### Summary
+Adds some exciting new features (listed below) for a long awaited release!
+
+### Features
+- Now add additional config variables with the `config_additional_variable` parameter!
+- Configure your management plugin with the new `collect_statistics_interval` parameter!
+- Enjoy more robust tuning of your TCP configuration with:
+  - `tcp_backlog`
+  - `tcp_sndbuf`
+  - `tcp_recbuf`
+
+### Bugfixes
+- MODULES-3740 - RabbitMQ template missing important config parameter
```

Comment:
Can you add that there was an update to the apt gpg key? This fix was a big reason for needing the release.